### ISSUE TITLE
Using GPU in the training  

### ISSUE THREAD

**Issue by RamadanHussein** [JaidedAI/EasyOCR#1320]
I'm training EasyOCR and noticed that the training is using the CPU while the machine has a GPU with tensors , any configuration hat can help to force the model to use the GPU 

![CPU vs  GPU](https://github.com/user-attachments/assets/d30537f3-8c46-4f59-8421-163d45ecf6d9)

**Comment by leeejuju** [comment]
you can run torch.cuda.is_available() to test gpu is available

**Comment by RamadanHussein** [comment]
Yes it is available however I noticed this behavior that it doesn't use the cuda , is this due to the torch version because I'm using 12.2 while my GPU card is RTX 4090

**Comment by romanvelichkin** [comment]
Are you using `trainer.ipynb` from this repo? Because it works for me. 

> is this due to the torch version because I'm using 12.2

You're probably talking about CUDA version, not torch version. I would recommend create python environment specifically for EasyOCR, because it will install proper pytorch and cuda versions it needs. Otherwise you could have your own versions or overwrite EasyOCR confing with different versions.

**Comment by TonyDewhurst10** [comment]
i got same issue on 12.8 cuda is that supported?
